### PR TITLE
G4

### PR DESCRIPTION
**Pull request by ricardopereira**

### CODE REVIEW

File: Spec/RestClient.swift [hunk 1]
```diff
@@ -19,6 +19,21 @@ class RestClient: QuickSpec {
...
+            // G4
+            pending("All REST requests should include the current API version") {
+                let client = ARTRest(options: AblyTests.commonAppSetup())
+                client.httpExecutor = testHTTPExecutor
+                let channel = client.channels.get("test")
+                waitUntil(timeout: testTimeout) { done in
+                    channel.publish(nil, data: "message") { error in
+                        expect(error).to(beNil())
+                        let version = testHTTPExecutor.requests.first!.allHTTPHeaderFields?["X-Ably-Version"]
+                        expect(version).toNot(beNil())
```
Comment (mattheworiordan) [inline Spec/RestClient.swift]: Why not check for an exact value?

**Comment by mattheworiordan** [comment]
Looks good, one comment

File: Spec/RestClient.swift (hunk 1 above)
Comment (ricardopereira) [inline Spec/RestClient.swift]: Done 519b433.

**Comment by tcard** [comment]
PTAL

**Comment by ricardopereira** [comment]
👍

**Pull request merged**